src/main/java/org/jabref/gui/menus/FileHistoryMenu.java

```diff
@@ -91,4 +101,9 @@ public void openFile(Path file) {
         }
         openDatabaseAction.openFile(file);
     }
+
+    public void clearLibrariesHistory() {
+        history.clear();
+        setDisable(true);
```

Comment:
This would disable the button when recent libraries are empty, but I don't see any code that would re-enable it when recent library item is added.

Reply:
I assume that `FileHistoryMenu#newFile(Path file)` [invocasion](https://github.com/JabRef/jabref/blob/main/src/main/java/org/jabref/gui/importer/actions/OpenDatabaseAction.java?plain=1#L159) within the `OpenDatabaseAction` class, will re-enable the item as the aforementioned method declares : `setDisable(false)`.

I also tested it manually and system's behavior is as expected. When we open a new library the `Recent libraries` menu will be correctly enabled and the recent libraries will be listed.